### DISCUSSION TITLE
NVD Data Feeds - On December 15th, 2023, the NVD plans to retire all legacy data feeds

### DISCUSSION THREAD

**Discussion by oliverlockwood** [aquasecurity/trivy#5652] [Q&A] [answered]
### Question

According to [https://nvd.nist.gov/vuln/data-feeds](https://nvd.nist.gov/vuln/data-feeds), on December 15th, 2023, the NVD plans to retire all legacy data feeds.

Would this affect Trivy in any way?

See also e.g. https://github.com/jeremylong/DependencyCheck/issues/6046.

### Target

Container Image

### Scanner

Vulnerability

### Output Format

None

### Mode

None

**Comment by knqyf263** [comment, marked as answer]
It will affect us. We saw a lot of people struggling with the new API, so we've been putting it off. We should migrate to NVD API 2.0 now. Created https://github.com/aquasecurity/trivy/issues/5658. Thanks.
